nav2_planner/src/planner_server.cpp

```diff
@@ -510,4 +517,24 @@ PlannerServer::publishPlan(const nav_msgs::msg::Path & path)
   }
 }
 
+void PlannerServer::isPathValid(
+  const std::shared_ptr<nav2_msgs::srv::IsPathValid::Request> request,
+  std::shared_ptr<nav2_msgs::srv::IsPathValid::Response> response)
+{
+  unsigned int threshold = 10;
```

Comment:
This seems like you should be using the pre-defined costmap types for LETHAL if just straight up checking if in collision now.